confgenerator/testdata/goldens/logging-processor_exclude_logs/golden/windows/fluent_bit_main.conf

```diff
@@ -514,7 +579,7 @@
     Match  p1.sample_logs
     Name   lua
     call   process
-    script 94fd0c547d4bfeb183bdafe62731e158.lua
+    script b154a21136f4178c508e555d6340f97f.lua
```

Comment:
welp, I don't know why this filter isn't being followed by a `modify` filter like I expect. I'm out of time tonight, will check into it tomorrow.